mypyc/primitives/int_ops.py

```diff
@@ -70,6 +70,12 @@
     error_kind=ERR_MAGIC,
     priority=3)
 
+size_t_to_short_int = c_custom_op(
+    arg_types=[c_pyssize_t_rprimitive],
+    return_type=short_int_rprimitive,
+    c_function_name='CPyTagged_ShortFromSsize_t',
+    error_kind=ERR_NEVER)
```

Comment:
As mentioned above, we don't want to use this in this PR.

Reply:
This can be removed, since it's unused.

Reply:
sure, I must have forgotten to do that.